### DISCUSSION TITLE
Necesito agregar un input para el documento de identidad

### DISCUSSION THREAD

**Discussion by manuog1994** [woocommerce/woocommerce#42942] [Checkout Flow]
He intentado inyectar código para crear el campo documento de identidad y me esta resultando imposible y mi facturación lo requiere, ¿Hay alguna manera de agregar este input? Actualmente estoy usando el Checkout por bloques con la plantilla Organic Goodness de Thunderstores en su documentación tampoco he encontrado nada relacionado con añadir campos al checkout.

**Comment by opr** [comment]
Hi there, thanks for starting this discussion - I don't speak Spanish so I used GitHub's translate feature to turn this to English - if anything in the translation is inaccurate please let me know, below is the translation that GitHub generated:

> I have tried to inject code to create the ID document field and it is proving impossible and my billing requires it, is there a way to add this input? I'm currently using the Block Checkout with Thunderstores' Organic Goodness template in their documentation, I haven't found anything related to adding fields to the checkout either.

If you're looking to add a new field then we are currently working on enabling this, however it will be some time before it is ready.

In the meantime you can consider adding a new block to the Checkout block - [here is a tutorial that should help you get started with that](https://developer.woo.com/2023/08/07/extending-the-woocommerce-checkout-block-to-add-custom-shipping-options/)

Please feel free to reopen this discussion if you need further help adding a block, and feel free to join the [WooCommerce Community Slack](https://woo.com/community-slack/) where a community of other Woo developers may be able to help you!

**Reply by manuog1994** [reply]
Hola, muchas gracias por su ayuda no estoy muy familiarizado con WooCommerce pero ese tutorial me salvo el día
